modules/data/pages/define-and-deploy-the-bdm.adoc

```diff
@@ -152,26 +152,35 @@ When you create a process that uses a business object with a composition or aggr
 
 [#lazy_eager_loading]
 
-=== Loading
+=== Loading BDM objects
 
-There are two options for loading complex business objects: *lazy* or *eager*. +
+There are two design patterns for loading business objects: *LAZY* or *EAGER*. +
```

Comment:
You should precise that it only concerns aggregation, if my memory is good

Reply:
what exactly? the design patterns or loading business objects?

Reply:
Your memory is NOT good, a business object used in composition or in aggregation can be loaded lazily or eagerly. 

But I do not recommend to use the _design pattern_ here. In computer science, a design pattern is a reusable solution to a commonly occurring problem.
The way to load a complex field has nothing to do with a design pattern IMHO. 
And you do not really load business objects, you load complex fields of a business objects.